diff --git a/src/GlobalHotKeys/NativeFunctions.cs b/src/GlobalHotKeys/NativeFunctions.cs
index 2317158..ed519b0 100644
--- a/src/GlobalHotKeys/NativeFunctions.cs
+++ b/src/GlobalHotKeys/NativeFunctions.cs
@@ -24,7 +24,7 @@ public partial class NativeFunctions
 
     [LibraryImport("user32.dll", EntryPoint = "CreateWindowExW", SetLastError = true, StringMarshalling = StringMarshalling.Utf16)]
     [UnmanagedCallConv(CallConvs = [typeof(System.Runtime.CompilerServices.CallConvStdcall)])]
-    public static partial IntPtr CreateWindowEx(int dwExStyle, uint regResult, string lpWindowName, WindowStyle dwStyle, int x, int y, int nWidth, int nHeight, IntPtr hWndParent, IntPtr hMenu, IntPtr hInstance, IntPtr lpParam);
+    public static partial IntPtr CreateWindowEx(int dwExStyle, uint regResult, string? lpWindowName, WindowStyle dwStyle, int x, int y, int nWidth, int nHeight, IntPtr hWndParent, IntPtr hMenu, IntPtr hInstance, IntPtr lpParam);
 
     [LibraryImport("user32.dll", SetLastError = true)]
     [UnmanagedCallConv(CallConvs = [typeof(System.Runtime.CompilerServices.CallConvStdcall)])]
